src/uint256.cpp

```diff
@@ -33,7 +33,7 @@ void base_blob<BITS>::SetHex(const char* psz)
         psz++;
 
     // skip 0x
-    if (psz[0] == '0' && tolower(psz[1]) == 'x')
+    if (psz[0] == '0' && ToLower(psz[1]) == 'x')
```

Comment:
Self-review:

Implicit conversion from `char` to `unsigned char` here. Should be handled explicitly?